Source/monster.h

```diff
@@ -316,6 +316,11 @@ struct Monster { // note: missing field _mAFNum
 		return *type().data;
 	}
 
+	bool IsAlive() const

```

Comment:
```suggestion
	bool isAlive() const
```

Reply:
Guess I looked at player functions first, they start with uppercase letters :D any idea why?

Reply:
I think it was because Source/.clang-tidy initially didn't define a style for MethodCase (class functions), so the fallback was FunctionCase. https://github.com/diasurgical/devilutionX/pull/4502 and https://github.com/diasurgical/devilutionX/pull/4493#discussion_r853613509 for the history